Editor/Scripts/Analytics/MeshsyncAnalytics.cs

```diff
@@ -48,12 +93,24 @@ private struct SyncEventData {
             public string entitySyncType;
         }
 
+
+        public MeshSyncAnalytics() {
+            this.syncEventStack = new Stack<TimedEvent<SyncEventData>>();
+            this.lastSync = DateTime.Now;
+
+            CoalesceEvents = true;
+            MinTimeBetweenSync = DEFAULT_TIME_SPAN;
+        }
+
         private static void logIfWarning(AnalyticsResult resp) {
-            if (resp != AnalyticsResult.Ok) {
+            if (resp != AnalyticsResult.Ok && resp != AnalyticsResult.TooManyRequests) {
```

Comment:
If the root of the problem here should fixed by the `CoalesceEvents` flag we should continue log the warning if it occurs rather than swallowing it, so we know if anything else causes it later on.

Reply:
The problem is that it's not clear what our users can do to remove/fix the warning when they see this, and it might actually bother them if they see a lot of these, so I would say we should actually remove the log warning.

Reply:
You got a point there, but like I mentioned above, users can't do anything to avoid having these logs when it happens now, and these logs aren't actually useful for them.
Therefore, these logs must be logged on the server side, not on the client.

Reply:
Hi Sindarta I don't believe it is possible to log errors like this on the server side with the analytics backend that is used. If you are indeed aware of a method of logging this on the server side I would be happy to take advantage of that instead.

Reply:
Well, that's something that we have to work together with the analytics team, isn't it ?
The other alternative is to try a different analytics backend.